lib/src/new-input-text/NewInputText.jsx

```diff
@@ -291,24 +287,26 @@ const DxcNewInputText = React.forwardRef(
 
     const decrementNumber = () => {
       const numberValue = value || innerValue;
```

Comment:
Wouldn't it be better: `const numberValue = value ?? innerValue;` ? If `value` has a [_falsy value_](https://developer.mozilla.org/en-US/docs/Glossary/Falsy), that includes 0, this variable would have the `innerValue` value instead, which i don't think is the right behaviour here, what do you think?

Reply:
In this case it is not totally necessary because value is a string (it is parsed to int when the changeValue function is called). So, in the case the value is "0", it will be correctly evaluated using the `||` operator. But I think it could be a good idea to change to `??` operator anyway.

Reply:
True, i was thinking as if it was a type number, I did not remember that it was a string, well thought!